docs/Tutorial.md

```diff
@@ -12,8 +12,7 @@ This 30 minutes tutorial will expose how to create a new admin app based on an e
 React-admin uses React. We'll use Facebook's [create-react-app](https://github.com/facebookincubator/create-react-app) to create an empty React app, and install the `react-admin` package:
 
 ```sh
-npm install -g create-react-app
-create-react-app test-admin
+npx create-react-app test-admin
```

Comment:
```suggestion
npx create-react-app test-admin
# or
yarn create react-app test-admin
# or
npm init react-app test-admin
```

Reply:
Reference: https://github.com/facebook/create-react-app#creating-an-app

Reply:
Didn't know about these extra commands. Thanks! That being said, I'm not sure we should be so exhaustive to keep it simple. But your choice! :)

Reply:
As a yarn user, I only knew about `npx` and `yarn create`. Just discovered npm had an `init` :)